ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
> By "now", I meant "after the data is migrated" ;)

But we don't know when it will be migrated on all servers, since we don't really have an upgrade policy (yet).

What I'm saying is that any talk of removing that model and what will happen when it is removed is premature at this point. I'm honestly seeing us keep it for the next few years, and I don't think it would be any problem at all.